nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -302,12 +307,39 @@ boolean symbolHasSuppressWarningsAnnotation(Symbol symbol, String suppression) {
     return false;
   }
 
-  static String errMsgForInitializer(Set<Element> uninitFields) {
+  static int GetLineforField(Element uninitField, VisitorState state) {
+    Tree fieldTree = getTreesInstance(state).getTree(uninitField);
+    DiagnosticPosition position = (DiagnosticPosition) fieldTree;
```

Comment:
I recommend adding a comment over this line saying that we expect `Tree` to be `JCTree` and thus implement `DiagnosticPosition` for statements in Java code. 

Also, in case that assumption ever breaks, do you guys think we should crash with a cast exception or just return `-1` or some other dummy value? My thoughts on this are: a) if we are just printing out the value, then returning `-1` makes this code more robust, but b) if we ever use this method for something else where we are acting on the line number info, then we could just be hiding the issue. I think I vote on leaving the code itself as is (with the comment), and just having it crash early if it ever sees a `Tree` it can't cast to `DiagnosticPosition`, but I thought it was worth to bring up the issue explicitly and get your thoughts on it @Lycheeicy and @msridhar :)

Reply:
I think we should just fail fast, which is what the current code will do.

Another possible failure mode is `getTreesInstance(state).getTree(uninitField)` returning `null`, e.g., if `uninitField` comes from bytecode.  We should add a null check and throw a `RuntimeException` with some appropriate message if that happens.

Reply:
Thanks for advice, I will add the comment as well as a null check later.